docs/blog/20220216-Daeraxa-ReleaseStrategyUpdate.md

```diff
@@ -0,0 +1,91 @@
+We also found that the very concept of the Cirrus CI binaries confused some people, particularly due to the name, so this also needed to be addressed.
+
+So after some lengthy discussions and the aforementioned poll, we decided to address these issues by reflecting what is actually going on in practice.
+
+So what does this mean for you? In terms of the application and releases themselves? Nothing. The biggest change is that the links and headers on our [downloads page](https://pulsar-edit.dev/download.html) have been reorganised and renamed to reflect how we want to go forward with our releases.
+The sections below will go into detail on each type.
+
+## Rolling Release
+
+The "Rolling Release" is the new name for what we previously called "Cirrus CI Binaries" and we have promoted this to the top of our downloads page to make it more obvious as we have a (soft) preference for its use - after all this is the "hyper-hackable text editor" and this seems to fit our mission statement nicely.
+
+This type of release has a number of benefits for everyone:
+
+- New functionality is available shortly after the relevant PR is merged. This means no waiting for a formal release milestone to include some desired features or fixes.
+- No formal release process is needed, we have a standard download link for each release binary that automatically pulls the very latest build from Cirrus CI.
+- Issues can be easily tracked to a particular PR or change because each version stamp is different for each PR so we can narrow down the problem rather easily.
+- Issues can be addressed quickly without the need for a formal tagged hotfix release process.
+
+Of course there are some potential drawbacks:
+
+- Despite our best efforts with reviews and testing it is entirely possible that things will fall through the cracks leading to a regression or issue meaning that you would need to downgrade or swap to the Regular Release until the issue gets resolved.
+- Whilst we don't have automatic updates implemented yet, when we do it means that an updated version will be almost constantly shown which might be seen as a curse or blessing from your standpoint.
+
+## Regular Releases
+
+This is the new name for the section previously just labelled as "Releases". These have been moved down on the page but remain a valid option for those that want to use a more regular release model.  
+Currently we are sticking to a release date of the 15th of every month (which initially was just a happy coincidence but we found we liked it and makes for easy organisation) but this not guaranteed and may change as needed.
+
+This also has some benefits for some over the rolling release:
```

Comment:
```suggestion
This also has some benefits for some, over the rolling release:
```

Reply:
I think this is technically correct already, at least from double checking with an online grammar checker, it didn't like the version with the comma.
https://quillbot.com/grammar-check

Reply:
Copy that then, just me reading it weird, I'll close this then

Reply:
There are some places in this and in #179 where I'd add commas, but I don't have the bandwidth to do that and the release dance, so I'm not worrying about it for now.